float32 deg_start
float32 deg_increment
float32 rad_increment 
float32[] measurements
float32[] bearings
